README.md

```diff
@@ -14,11 +13,11 @@ Also install
 
 For downloading opensource projects use:
 
-`repo init -u git@gitpct.epam.com:rec-inv/android_manifest.git -m doma.xml -b android-11-master`
+`repo init -u git@github.com:svlad-90/android_manifest.git -m doma.xml -b android-14-prod-cockpit-rcar-v4h`
```

Comment:
Hi @arminn, please fix the repository link to point to this repo and not to my fork.